### DISCUSSION TITLE
Has something changed? Aug 2023 attempt to create site fails.

### DISCUSSION THREAD

**Discussion by johngage** [kevinlin1/just-the-class#34] [Show and tell]
Hi, I see many beautiful sites, but after creating a new repository, modifying the .yaml, and generating pages, I cannot generate a site that looks like [just-the-class](https://kevinl.info/just-the-class/).

Here is the generated site: no menu, no index.html: https://johngage.github.io/InfrastructureChat/

Am I missing something simple?

Thank you for the template.

**Comment by kevinlin1** [comment]
Hi John,

Thanks for posting! I took a look at your repository and I think a few changes to the `url` and `baseurl` can help. Here's what I think the lines should look like.

```
baseurl: '/InfrastructureChat'
url: 'https://johngage.github.io'
```

**Reply by johngage** [reply]
Kevin, thanks...and you were on it instantly, amazing.
